### PR TITLE
Update audit-log-retention-policies.md

### LINKED ISSUE #11654: Contradicting statements about audit log retention period based on user license?
Within the **[Default audit log retention policy](https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-retention-policies?view=o365-worldwide#default-audit-log-retention-policy)** section, the note suggests that non-E5 or guest users will only have 90-day retention of audit events. 

`The default audit log retention policy only applies to audit records for activity performed by users who are assigned an Office 365 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. If you have non-E5 users or guest users in your organization, their corresponding audit records are retained for 90 days.
`

However, in the note within the **[Before you create an audit retention policy](https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-retention-policies?view=o365-worldwide#before-you-create-an-audit-log-retention-policy)** section right below it, it is suggested that for users who do not meet licensing requirements, the data may be retained based on the highest priority retention policy. 

`If the user generating the audit log doesn't meet these licensing requirements, data is retained according to the highest priority retention policy. This may be either the default retention policy for the user's license or the highest priority policy that matches the user and its record type.`

Can you please clarify what the retention behavior is for such users who have non-E5 or guest licenses but are acting on services that are covered within an audit retention policy? E.g. - if there is an M365 E3 licensed user in SharePoint, will their audit events be retained only for 90-days as they do not meet Audit (Premium) licensing, or will the events be retained for 1-year since SharePoint events are retained for that duration for all users/admins as part of default retention policy?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e52e1877-191e-5dbe-b805-e755c736b985
* Version Independent ID: 69df5ed6-a86f-06d7-1156-e1de211f8e83
* Content: [Manage audit log retention policies - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-retention-policies?view=o365-worldwide#default-audit-log-retention-policy)
* Content Source: [microsoft-365/compliance/audit-log-retention-policies.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/audit-log-retention-policies.md)
* Service: **o365-seccomp**
* GitHub Login: @robmazz
* Microsoft Alias: **robmazz**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the note 

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11654

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/audit-log-retention-policies.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ You can create and manage audit log retention policies in the Microsoft Purview
...
 
 > [!NOTE]
-> The default audit log retention policy only applies to audit records for activity performed by users who are assigned an Office 365 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. If you have non-E5 users or guest users in your organization, their corresponding audit records are retained for 90 days.
+> The default audit log retention policy only applies to audit records for activity performed by users who are assigned an Office 365 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. If you have non-E5 users or guest users in your organization, their corresponding audit records are retained for 90 days or the highest priority policy that matches the user and its record type.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/audit-log-retention-policies.md]:
```suggestion
> The default audit log retention policy only applies to audit records for activity performed by users who are assigned an Office 365 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. If you have non-E5 users or guest users in your organization, their corresponding audit records are retained for 90 days or for the highest priority policy period that matches the user and their record type.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
closing as per the comment in #11654

**Pull request closed without merging**